### PR TITLE
Implement new layout, from design system

### PR DESCRIPTION
**Pull request by dherault**
Resolves ENG-15
## Summary

- Implement the new Sidebar from the design-system lib
- Implement the new Topbar
- Work on menus

## Checklist

- [x] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

**Comment by dherault** [comment]
Important note:
I spent the last hour talking with Nick about the Design of the app, he will propose changes that can nihilate the continuation of this work. 
Hence I'm stopping mid-way.
This is an improvement though.

Bugs left to fix:
- The overflow of the layout.

**Review by dogmar — APPROVED** [review]

File: www/src/components/layout/WithNotifications.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  console.log('notification', notification)
+  const { actor, type, repository, incident, insertedAt } = notification
+
+  function getUrl() {
+    if (type === 'LOCKED') return `/repository/${repository.id}`
+
+    return `/incident/${incident.id}`
+  }
+
+  function renderType() {
```
Comment (michaeljguarino) [inline www/src/components/layout/WithNotifications.js:97]: should this be an inner function?

File: www/src/components/layout/WithNotifications.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+      )}
+    </Flex>
+  )
+}
+
+function Notification({ notification, closePanel }) {
+  console.log('notification', notification)
+  const { actor, type, repository, incident, insertedAt } = notification
+
+  function getUrl() {
```
Comment (michaeljguarino) [inline www/src/components/layout/WithNotifications.js:91]: why is this not just `const url = type === 'LOCKED' ? '/repository/${repository.id}` : ...`

**Comment by michaeljguarino** [comment]
Also not sure this is being done, but there's a subscription for notifications that needs to be live for you to update the badging/notif list in real-time

File: www/src/components/layout/WithNotifications.js (hunk 2 above)
Comment (dherault) [reply www/src/components/layout/WithNotifications.js:91]: Haha I copy pasted your code 😛

File: www/src/components/layout/WithNotifications.js (hunk 1 above)
Comment (dherault) [reply www/src/components/layout/WithNotifications.js:97]: I find it clearer this way.

**Comment by dherault** [comment]
It's on my todo list.

**Pull request merged**